### PR TITLE
doc: how to test filters

### PR DESCRIPTION
**Pull request by tommyblue**
#### :question: What

how to test filters

### CODE REVIEW

File: content/en/docs/How Tos/create_filter.md [hunk 1]
```diff
@@ -209,3 +209,44 @@ func (f *MyFilter) Process(r Record, next func(Record)) {
...
+
+if filter, err := NewMyFilter(cfg); err != nil {
+    t.Errorf("unexpected error: %v", err)
+}
+```
+
+> Obviously, if the filter requires some configuration values (not like this empty demo filter),
+the test should also test all possible values and corner cases.
+
+With the `filter` instance, is then possible to test the `Process()` function, providing a
```
Comment (arl) [inline content/en/docs/How Tos/create_filter.md:240]:
```suggestion
With the `filter` instance, it's then possible to test the `Process()` function, providing a
```

**Review by arl — APPROVED** [review]
Some comments/propositions.

For how-tos, guides or tutorials kind of docs, I like the user of _we_, to involve the reader, like you're showing them the way to do stuff, with them following along. But it's probably a matter of personal taste, just wanted to give my 2c about that :-)
